tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27
+envlist = py35,py36
```

Comment:
I think we are targeting to deploy this in Openshift using the centos python source-to-image which currently only have 3.6. But I agree we should make sure the tests are also running on 3.7

Reply:
My concern is also getting the RPM package updated in Fedora.

Reply:
Do we really need it packaged in Fedora ? or should we just retire it ?

Reply:
I've asked to retire it in the first place. https://bugzilla.redhat.com/show_bug.cgi?id=1627257

Reply:
@sayanchowdhury what do you think about retiring this package ? we are starting to run more and more application by just using pip or source-2-image, I think that would also work for fedbadges.

Reply:
Sure, I'm okay to retire but I don't have much knowledge about how much effort would it require to migrate the infra to OpenShift. 

Also, this PR can be halted till we don't have migration PR in `tahrir`

Reply:
I'd really rather not see packages getting retired like this, since it makes it harder for other people to deploy and otherwise reuse our applications. 😢